### PR TITLE
#50 - Script ex1

### PR DESCRIPTION
**Pull request by FatCaterpillar**
Decided to go with a script that enables to navigate between exercises instead of only going to next. In this case we can just tell the people to goToExercise x instead of do nextExercise twice.

### CODE REVIEW

File: goToExercise.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+EX="$1"
+
+if [ -z  "$EX" ]
+then
+  print_usage_and_exit
+else
+  git add .
+  git commit -m "Some work"
+  git checkout "ex${EX}"
```
Comment (krzykrucz) [inline goToExercise.sh:18]: git checkout origin "ex${EX}" -b "ex${EX}"

and please commit that to all branches :)

**Review by krzykrucz — CHANGES_REQUESTED** [review]

File: goToExercise.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+EX="$1"
+
+if [ -z  "$EX" ]
+then
+  print_usage_and_exit
+else
+  git add .
+  git commit -m "Some work"
```
Comment (jakubiec) [inline goToExercise.sh:17]: maybe message could be an optional argument to the script?

Comment (FatCaterpillar) [reply goToExercise.sh:17]: I don't think that's necessary, I'd leave it as it is :)

File: goToExercise.sh (hunk 1 above)
Comment (FatCaterpillar) [reply goToExercise.sh:18]: This fragment does not work. What's wrong with current way?

**Pull request merged**